docs/source/guide/Docker_README.md

```diff
@@ -103,6 +103,13 @@ Available types:
 * LocalKeyVault (local file system)
   - Requires variable ``RSTUF_LOCAL_KEYVAULT_PATH``
     - Define the directory where the data will be saved, example: `keyvault`
+  - Optional variables:
+    - ``RSTUF_LOCAL_KEYVAULT_ONLINE_KEY_NAME``
+      - file name of the online key
```

Comment:
```suggestion
      - file name of the online key. Default: `online.key`
```
In case the user decides to rename the key to the default value and not use a specific one.